scripts/local_cloudbuild.py

```diff
@@ -274,7 +288,9 @@ def local_cloudbuild(args):
     # Run shell script
     if cloudbuild.run:
         args = [os.path.abspath(cloudbuild.output_script)]
-        subprocess.check_call(args)
+        return subprocess.check_output(args)
```

Comment:
The output is no longer being shown in the console as the script runs due to change to `check_output` from `check_call`.